src/SDKs/Reservations/Reservations.Tests/Reservations.Tests.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Reservations.Tests</PackageId>
+    <Description>Reservations.Tests Class Library</Description>
+    <Authors>Microsoft Corporation</Authors>
+    <AssemblyName>Reservations.Tests</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp1.1</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Microsoft.Azure.Management.Reservations\Microsoft.Azure.Management.Reservations.csproj" />
```

Comment:
you are still missing ResourceManager 1.6.0 (latest version)
You are using the implicitly added reference to record all your tests. That version is 1.1.0 which is really old and was published last year. You need to use the latest version.

Reply:
latest version added

Reply:
I actually think this reference is not used in the project, I removed the reference and build succeeded and test all passed